quickwit-cli/src/cli.yaml

```diff
@@ -12,17 +12,24 @@ subcommands:
                 about: Creates an index.
                 display_order: 1
                 args:
-                    - metastore-uri:
-                        about: Location of the metastore.
-                        long: metastore-uri
-                        value_name: METASTORE URI
+                    - index-id:
+                        about: ID of the target index.
+                        long: index-id
+                        value_name: INDEX ID
                         required: true
-                        env: QUICKWIT_METASTORE_URI
                     - index-config-uri:
                         about: Location of the index config file.
                         long: index-config-uri
                         value_name: INDEX CONFIG URI
                         required: true
+                    - index-uri:
```

Comment:
do we have a default for the index-uri?

Reply:
Also could we put that in the index config file?

Reply:
yes there is but not very explicit though. Adding a default_index_root_uri in the config file sound not bad, it has the merit of saying "ok if you don't set the URI of your index, it will be at `default_index_root_uri/{index-id}`.

Reply:
I add `default_index_root_uri `.